docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3075.md

```diff
@@ -0,0 +1,29 @@
+# Audit pipe mount operations
+
+Test verifies that Pipe mount read, write delete operations are monitored and logged.
+
+**Prerequisites**:
+- Admin user
+- Non-admin user
+- perform [3059_2](3059_2.md) case
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the non-admin user from the prerequisites | |
+| 2 | At the Runs page, click the run launched at step 4 of [3059_1](3059_1.md) case | |
+| 3 | Wait until the SSH hyperlink appears | |
+| 4 | Click the SSH hyperlink | |
+| 5 | In the opened tab, enter and perform the `echo "test info" >> cloud-data/storage2/file3` command | |
+| 6 | Logout and login as the admin user from the prerequisites | |
+| 7 | Open the **Settings** page | |
+| 8 | Open the **SYSTEM MANAGEMENT** page | |
+| 9 | Select the **LOGS** tab | |
+| 10 | Click ***Show advanced*** button | |
+| 11 | Click **Type** field and select `audit` value from drop-down | Record <br> `WRITE storage2/file3` <br> is shown in the top of logs records |
+| 12 | Logout | |
+| 13 | Repeat steps 1-12 for `cp -rf cloud-data/storage2/folder1 cloud-data/storage1/folder1` command | For each files from folder1 (*inner_file1* and *inner_file2*) records <br>`READ storage2/folder1/<file>` <br> `WRITE storage1/folder1/<file>` <br> are shown in the top of logs records |
+| 14 | Repeat steps 1-12 for `mv -f cloud-data/storage2/folder2 cloud-data/storage1/folder2` command | For each files from folder2 (*inner_file3* and *inner_file4*) records <br> `READ storage2/folder2/<file>` <br> `WRITE storage1/folder2/<file>` <br> `DELETE storage2/folder2/<file>` <br> are shown in the top of logs records |
+| 15 | Repeat steps 1-12 for `rm -f cloud-data/storage2/file2` command | Record <br> `DELETE storage2/file2` <br> is shown in the top of logs records |
+
+**After:**
+- Stop run launched at step 4 of [3059_2](3059_2.md) case
```

Comment:
Add the item:
"- Delete storages created at step 2 in the Preparations of [3059_1](3059_1.md) case"